pom.xml

```diff
@@ -77,6 +78,14 @@
 			<artifactId>geojson-jackson</artifactId>
 			<version>${geojson-jackson.version}</version>
 		</dependency>
+		<!-- Needed until dept44 updates to at least 3.25 of this dependency -->
```

Comment:
why?

Reply:
To be able to use "withEnumStringComparison()" in tests. I'll put that in the comment as well